website_product_configurator/controllers/main.py

```diff
@@ -615,7 +609,7 @@ def get_attr_classes(self, attr_line, attr_value=False, custom=False):
         if attr_line.required:
             classes.append('required')
 
-        if attr_value and not product_tmpl.value_available(attr_value.id):
+        if attr_value and not product_tmpl.values_available([attr_value.id]):
```

Comment:
Hi @richard-willdooit,
The "values_available" method can be called and used like the "value_available" method by wrapping the first argument in a list.